db/migrate/20220603120651_create_comments.rb

```diff
@@ -0,0 +1,9 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.integer :author_id
+
+      t.timestamps
```

Comment:
- [ ] They should be a column for adding comments, that is `t.text :text` Please let's include this column.